src/frontend/komponenter/App.tsx

```diff
@@ -20,6 +20,9 @@ const App: React.FC = () => {
         });
     }, []);
 
+    if (!innloggetSaksbehandler) {
```

Comment:
Ikke helt sikker på denne, for de som ikke har tilgang til tjenesten så kommer de bare få en vit melding nå uten noen informasjon. Men er nok like greit